server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailSubmissionSetMethodFutureReleaseContract.scala

```diff
@@ -712,6 +744,7 @@ trait EmailSubmissionSetMethodFutureReleaseContract {
       .body(request)
       .post
 
+    Thread.sleep(5000)
```

Comment:
Can we find other solutions than such a looooong wait time? Why do we need to wait for?